rest/model/task.go

```diff
@@ -537,6 +537,9 @@ func (at *APITask) ToService() (*task.Task, error) {
 	}
 
 	if at.StepbackInfo != nil {
+		if st.StepbackInfo == nil {
```

Comment:
is there a reason why we dont do something like this here?
`st.StopbackInfo = stepbackinfo{
  lastfailing 
  lastpassing
...
}`